reference/release-notes/1.24.0.md

```diff
@@ -0,0 +1,105 @@
+---
+orphan: true
+---
+# 1.24.0
+
+These release notes cover new features and changes in Anbox Cloud 1.24.0.
+
+Anbox Cloud 1.24.0 is a minor release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+### Updates to components
+
+* Anbox Cloud includes and uses the following latest versions for components it uses:
+  - LXC version 6.0.2<!--AC-2812-->
+  - Mesa driver version 24.2.6<!--AC-2800-->
+  - VirGL renderer version 1.9.0.<!--AC-2800-->
+
+### Security
+
+* We have enhanced security with a minimum version of TLS 1.3 as a requirement for all services working with Anbox Cloud. TLS 1.3 is also enabled by default for the Anbox Stream Agent and Gateway.<!--AC-2728 and AC-2654-->
+
+### Anbox Management Service (AMS)
+
+* We have increased the maximum number of queued operations in AMS to allow better performance.<!--AC-2831-->
+* Anbox Cloud provides the image type (`android`, `aaos` or `generic`) through the following three ways:<!--AC-2757-->
+  - `metadata.yaml` of the Anbox Cloud image
+  - API response from the AMS when querying image information
+  - Output of `amc image show` command
+
+  This allows the web client to adapt its interface according to the image type.
+* We have enabled support for multiple cluster management by AMS.<!--AC-2674-->
```

Comment:
This is not accurate.

The story is about enabling dashboard to communicate with multiple AMS instances.
However when I looked at the AC-2674, I didn't find a PR yet. So I am not quite sure if this feature is enabled or not in this cycle.
@jat-canonical  could you clarify on this as well?
Thanks